src/common/schemas/json-schema.service.ts

```diff
@@ -8,6 +9,7 @@ export class JsonSchemaService {
   constructor() {
     // coerceTypes param shouldn't be necessary when serialization is implemented.
     this.ajv = new Ajv({ coerceTypes: true });
+    addFormats(this.ajv, { formats: ['uri'] });
```

Comment:
This adds support for additional checks on AJV. In this case it adds support for checking `uri`.